account_sale_stock_report_non_billed/models/res_company.py

```diff
@@ -10,3 +10,4 @@ class ResCompany(models.Model):
     stock_move_non_billed_threshold = fields.Date(
         string="Non Billed Threshold Date", default=fields.Date.today
     )
+    use_dates_as_deadlines = fields.Boolean()
```

Comment:
This should be put at wizard level, not at company level.

Reply:
And a better name is: `interval_restrict_invoices`

Reply:
Okay, I will change the name, I just put this at company level to allow set this value as default if the client wants. Obviusly, it can be changed at wizard level

Reply:
OK, not bad idea, but please use the existing res.config.settings mechanism for that: putting `default_` + field name + an attribute indicating the model, you make this automatic using default values. Not sure if it can be company related though.

Reply:
I didn't know about this mechanism! Thanks for the info :smile: